zbench.zig

```diff
@@ -218,16 +201,16 @@ pub const BenchmarkResults = struct {
 
 pub fn run(comptime func: BenchFunc, bench: *Benchmark, benchResult: *BenchmarkResults) !void {
     defer bench.durations.deinit();
-    const MIN_DURATION = 1_000_000; // minimum benchmark time in nanoseconds (1 millisecond)
-    const MAX_N = 10000; // maximum number of executions for the final benchmark run
-    const MAX_ITERATIONS = 10; // Define a maximum number of iterations
+    const MIN_DURATION = 1_000_000_000; // minimum benchmark time in nanoseconds (1 second)
+    const MAX_N = 65536; // maximum number of executions for the final benchmark run
```

Comment:
What is the reason for `MAX_N` and `MAX_ITERATIONS` to be so specific?

Reply:
Those are powers of 2, no specific reason, sorry for the confusion ;-) The numbers are basically orders of magnitude where I found the benchmarks to run smoothly, and to not show too much variance.
I was thinking if we might give them reasonable defaults but expose as configurable, to find optimal settings for a specific scenario?